user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md

```diff
@@ -0,0 +1,92 @@
+
+To configure this component:
+
+1. Apply the necessary data feeds. See [Applying a data feed](xref:Configuring_dashboard_components#applying-a-data-feed).
+
+   > [!NOTE]
+   > The assigned feed name will be visible in the bottom-right corner of the query filter component (e.g. Query filter 1).
+
+1. Optionally, fine-tune the component layout. See [Customizing the component layout](xref:Configuring_dashboard_components#customizing-the-component-layout).
+
+1. Optionally, customize the following component options in the *Component* > *Settings* tab:
+
+   - *WebSocket settings*: Allows you to customize the polling interval for this component. To do so, clear the checkbox in this section and specify the custom polling interval.
+
+   - *Filter assistance*: Determines whether the column filters will assist you by providing suggestions and other information such as units, step size, etc. By default disabled.
+
+     > [!NOTE]
+     > Depending on the size of your data, enabling this setting may impact performance.
+
+   - *Allow color mode*: Determines whether [conditional coloring](#conditional-coloring) can be applied within the query filter. By default enabled.
+
+- In the top-right corner of the query filter component, you have an *Active (x)* toggle button. If you enable this button, the component displays only the active filter options, and the button itself indicates the number of active options.
+
+- Next to columns containing discrete values of type string or number, you can find a button that allows you to change how the possible values are displayed when you hover your mouse over it:
+
+  - Select the *Toggle free form* button to display a text box in which you can type a value.
+
+  - Select the *Toggle checklist* button to list all possible values as a checklist.
+
+- When exclusively filtering a node edge graph by node, edges will only be highlighted when both the source and destination are highlighted. Alternatively, when exclusively filtering a node edge graph by edge, the source and/or destination attached to the highlighted edge segments will be highlighted.
+
+## Filtering a table or node edge graph component
+
+When linked to a component using a feed, the query filter component allows real-time filtering of that associated component.
+
+Two methods exist for linking a query filter:
+
+- [Feeding queries as data](#feeding-queries-as-data)
+
+- [Feeding query columns as filter](#feeding-query-columns-as-filter)
+
+### Feeding queries as data
+
+Following setup and configuration of the query filter component:
+
+1. Add a table component to your dashboard.
+
+   > [!NOTE]
+   > In this example, we are using a table component. However, any component capable of receiving a query can be linked to the query filter.
```

Comment:
I have added this disclaimer so that it is clear that the table component is only one example of a component that can be linked to a query filter, @TVincke98.